languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -19,7 +19,7 @@ When evaluating objects in `if` statements or other boolean contexts, all object
 
 ## Control flow
 
-_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct, but it may be more common to use `if` as a "guarding" statement to modify the evaluation of an expression.
+_Truthy_ and _falsey_ evaluations are useful in the context of control flow.
```

Comment:
[Lines 33-54](https://github.com/exercism/v3/blame/83b6a40174d0db1ff3fb4681c00eb860a8be1219/languages/ruby/exercises/concept/booleans/.docs/introduction.md#L33-L54) still contain if and unless

Reply:
@neenjaw So we don't want nothing connected to `if/else` here? Yup?

Reply:
Yeppers, so it makes it kind of tricky.  All we need to give is _just enough_ information for the student to go on so that they can google it further.  And since `if/else/unless` aren't required for this exercise, they should be removed.

:tada: